src/manifold/utils/data-processor.js

```diff
@@ -205,18 +202,13 @@ export function columnsAndFieldsFromX(x) {
   const xFieldNames = Object.keys(x[0]);
   xFieldNames.forEach(fieldName => {
     const featureData = x.map(row => row[fieldName]);
-    if (!isFeatureInvalid(featureData)) {
+    const field = computeFeatureMeta(fieldName, featureData);
+    if (field.type !== 'null') {
```

Comment:
should we add a constant for null field type since it's marked as a string?

Reply:
This should actually be `null` (not a string)